docs/internal/using-the-nuget-packages.md

```diff
@@ -41,6 +41,15 @@ the `OpenTelemetry.AutoInstrumentation` package to your project:
 dotnet add [<PROJECT>] package OpenTelemetry.AutoInstrumentation --source <PATH_TO_AUTO_INSTRUMENTATION_PACKAGES> --prerelease
 ```
 
+If the application references packages that can be instrumented, but, require
+other packages for the instrumentation to work the build will fail and prompt
+you to either add the missing instrumentation package or to skip the
+instrumentation of the corresponding package:
```

Comment:
I'd add an example how to skip the instrumentation (especially for scenario when instrumentation of multiple packages is to be skipped, where trying to separate package names with comma or semicolon would cause errors)

Reply:
4396cfd1f808110481c627e8733e9f49935078d7